modules/ting_das/css/ting_das.css

```diff
@@ -0,0 +1,79 @@
+.popupbar .popupbar-container {
```

Comment:
@yepzdk will you comment on this.

Reply:
You are right. We would prefer to use BEM/SMACSS. But in this situation we are reusing the popupbar component and want it to behave different in some aspects, we needed to override some of the already existing styling and therefore be more specific to ensure the overrides.

Reply:
A refactor of the existing component is out of scope for this. And for the same reason we are working in plain css.

It could probably be done, but then we would need to use the !important statement even more.
So I preferred to keep .popupbar in front to be explicit enough for our overides to work.